packages/fluentui/accessibility/src/behaviors/Menu/menuItemBehavior.ts

```diff
@@ -39,7 +37,6 @@ export const menuItemBehavior: Accessibility<MenuItemBehaviorProps> = props => (
       'aria-labelledby': props['aria-labelledby'],
       'aria-describedby': props['aria-describedby'],
       'aria-disabled': props.disabled,
-      [IS_FOCUSABLE_ATTRIBUTE]: !props.disabled,
```

Comment:
Should we remove the attribute or should we have:

`[IS_FOCUSABLE_ATTRIBUTE]: !props.disabled`?